src/Arcus.WebApi.Unit/Security/Authentication/SharedAccessKeyAuthenticationAttributeTests.cs

```diff
@@ -14,38 +14,41 @@ public class SharedAccessKeyAuthenticationAttributeTests : IDisposable
     {
         private const string HeaderName = "x-shared-access-key",
                              SecretName = "custom-access-key-name",
-                             AuthorizedRoute = "/authz/shared-access-key";
+                             AuthorizedRoute = "/authz/shared-access-key",
+                             ParameterName = "api-key";
 
         private readonly TestApiServer _testServer = new TestApiServer();
 
         [Theory]
-        [InlineData(null, "not empty or whitespace")]
-        [InlineData("", "not empty or whitespace")]
-        [InlineData(" ", "not empty or whitespace")]
-        [InlineData("not empty or whitespace", null)]
-        [InlineData("not empty or whitespace", "")]
-        [InlineData("not empty or whitespace", " ")]
+        [InlineData(null, null, "not empty or whitespace")]
+        [InlineData("", "", "not empty or whitespace")]
+        [InlineData(" ", " ", "not empty or whitespace")]
+        [InlineData("not empty or whitespace", "not empty or whitespace", null)]
+        [InlineData("not empty or whitespace", "not empty or whitespace", "")]
+        [InlineData("not empty or whitespace", "not empty or whitespace", " ")]
         public void SharedAccessKeyAttribute_WithNotPresentHeaderNameAndOrSecretName_ShouldFailWithArgumentException(
```

Comment:
Test name needs to be aligned
```suggestion
        public void SharedAccessKeyAttribute_WithNotPresentHeaderNameQueryParameterNameAndOrSecretName_ShouldFailWithArgumentException(
```